src/FreeBirdIO.jl

```diff
@@ -64,10 +64,12 @@ function convert_system_to_walker(at::FlexibleSystem, resume::Bool)
     data = at.data
     energy = (haskey(data, :energy) && resume) ? data[:energy]*u"eV" : 0.0u"eV"
     iter = (haskey(data, :iter) && resume && data[:iter]>=0) ? data[:iter] : 0
-    num = (haskey(data, :num_frozen_part)  && resume) ? data[:num_frozen_part] : 0
+    list_num_par = (haskey(data, :list_num_par) && resume) ? data[:list_num_par] : [length(at)]
```

Comment:
When reading in an .extxyz file, if the file already contains these "keys" and we want to resume the calculation, they will be restored. Otherwise, each of them gets reassigned a new starting value.

Reply:
For example:
```julia
energy = (haskey(data, :energy) && resume) ? data[:energy]*u"eV" : 0.0u"eV"
```
is equivalent to:
```julia
if haskey(data, :energy) && resume # both true
    energy = data[:energy]*u"eV" # read in the data
else
    energy = 0.0u"eV" # assign 0 eV as the new starting energy
end
```